memorystore/redis/cloud_run_deployment/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Use the official Golang image to create a build artifact.
+# This is based on Debian and sets the GOPATH to /go.
+# https://hub.docker.com/_/golang
+FROM golang:1.13 as builder
```

Comment:
Should we use 1.13 or 1.14? If this is only for Cloud Run, it seems like we should use 1.14?

Reply:
The current "canonical" Dockerfile base image is `golang:1.13-buster`, though #1541 proposed to update that to 1.14. I'm fine rolling that forward. More significantly, this Dockerfile is a copy of an earlier version of the template used in Cloud Run, it would be best to update this from [the Cloud Run Pub/Sub sample Dockerfile](https://github.com/renovate-bot/golang-samples/blob/renovate/docker-all/run/pubsub/Dockerfile)

Reply:
I've updated the Dockerfile based on the Cloud Run Pub/Sub sample, thanks